src/library/models.py

```diff
@@ -705,17 +711,27 @@ def get_reading_data(cls, period: Period, days=0, sort='name', username=None):
             result.sort(reverse=True, key=lambda item: item['book_count'])
         elif sort == StudentActivitySort.TIME:
             result.sort(reverse=True, key=lambda item: item['hours'])
+        
+        if (books_sort):
+            for reading_data_for_one_user in result:
+                if books_sort == ReadingDetailsSort.TITLE:
+                    reading_data_for_one_user['books'].sort(key=lambda item: item['title'])
```

Comment:
Better would be `sort_title` rather than `title`.

Reply:
@bgoldowsky could you clarify which `title` would be better to be `sort_title`? Do you mean change `ReadingDetailsSort.TITLE` to ReadingDetailsSort.SORTTITLE`? I don't think the `title` in the variable `item['title']` can be changed as it's the field name defined in the `Book` model. Thanks.

Reply:
Sorry. `Book` has both fields - I meant it would be better to sort on `Book.sort_title` rather than `Book.title`.

Reply:
Got it. Will make the change. Thanks.